### ISSUE TITLE
[med] Chrome + Jaws: Screen reader announces extra information on text fields.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8617]
## Action Performed:
1. Using JAWS + Chrome browser, open the URL staging.new.expensify.com/r/85021985     
2. Open settings and navigate to the workspace using Tab key.
3. Activate workspace using Enter key and Double click on the help button.   
4. New content will appear, navigate to the request a setup call using Tab key and press Enter.
5. Navigate the text field using Tab key and listen to the screen reader announcement.

## Expected Result:
Only the information of a text field should be announced. it should read as "first name text field"

## Actual Result:
Screen reader announce extra information of the text fields that is not visually present page.                                                      Screen reader announced as "New Expensify first name text field".

HTML
`<input autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" spellcheck="true" type="text" class="css-11aywtz r-13awgt0 r-tz77pi r-13qz1uu" value="" name="fname" style="border-radius: 8px; border-width: 0px; box-shadow: none; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; outline-width: 0px; padding: 23px 11px 8px;">`

**Other occurrences**
>Same issue also repro on PR #8868, #8802
>Same issue repro on PR #8769 -Extra information announced on conceirge control.

## Workaround:
Yes

## Area issue was found in:
Workspace -> Get assistance

## Failed WCAG checkpoints
1.3.1

## User impact:
Screen reader users might be confused by the announcement of more information and about where the current focus.

## Suggested resolution:
Make sure the screen reader announce only the information of element visually present on the page.                                                                           Refer to:              
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

## Platform:

- Web

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/163122959-dc9887e2-f3f2-486b-9f63-b8fb09e3907c.mp4
